### DISCUSSION TITLE
Stockfish

### DISCUSSION THREAD

**Discussion by jc256cbcs** [official-stockfish/Stockfish#5569] [General]
I'm an old, relatively computer-ignorant person who would like to use Stockfish. I downloaded the file, extracted the files, and ran it on my PC, but it came up without a GUI. So, I tried to download a GUI (lichens) for the engine and cannot get it to load. Anyone out there who can explain all of this in a format that a five-year-old could understand would be much appreciated.

**Comment by Disservin** [comment]
Dunno about a lichens gui, try downloading one from here and then follow the install process further down the page.
https://github.com/official-stockfish/Stockfish/wiki/Download-and-usage#download-a-chess-gui

**Comment by jc256cbcs** [comment]
Thanks so much.  Got it downloaded. loaded the engine and started a game,
but I don't really know what I'm doing yet.  Thanks again.

John

**Comment by jc256cbcs** [comment]
BTW - how is this any different than just playing online with a chess
engine such as chess.com?  Confused.

**Reply by dav1312** [reply]
You can see some of the reasons here in the note about online engines https://github.com/official-stockfish/Stockfish/wiki/Download-and-usage#online
